Fineo/Common/Fineo.Interfaces/IMessageBus.cs

```diff
@@ -18,10 +18,14 @@ public interface IMessageBus
     {
         void Init(IMessageBusParams initParams);
 
+        string ID { get; set; }
```

Comment:
Why this one is needed?

Reply:
Uniquely identifies client. In case if ID = receiver_id in the message, message will be automatically removed from the queue